config/sentry.php

```diff
@@ -10,6 +10,9 @@
     // @see https://docs.sentry.io/product/sentry-basics/dsn-explainer/
     'dsn' => env('SENTRY_LARAVEL_DSN', env('SENTRY_DSN')),
 
+    // @see https://spotlightjs.com/
+    'spotlight' => env('SENTRY_SPOTLIGHT', false),
+
```

Comment:
Let's treat this like the logger, as we don't want this to be enabled on prod.
So I would suggest no env var and commented out.

Reply:
It's up to you, but not having an `.env` means you _have_ to publish the config files even though almost everything else is configurable without the config file published (it would be best if we wouldn't need users to publish the config file at all). Just add's an extra step to enable it locally, but no super hard feelings either way, so up to you.

Reply:
So far we told users during on-boarding to publish their config.
Let's revisit this in the next major, though there might be some surprises when we add new things to the SDK config and people get it automatically.

You can leave it as is then.